jetstream/workflows/run.yaml

```diff
@@ -48,6 +48,11 @@ spec:
         "--dataset_id={{workflow.parameters.dataset_id}}", 
         "--project_id={{workflow.parameters.project_id}}"
       ]
+      resources:
+        requests:
+          memory: 8Gi   # make sure there is at least 8Gb of memory available for the task
+        limits:
+          cpu: 4  # limit to 4 cores
```

Comment:
This should be enough for most tasks, but can be tweaked if we see any performance issues. For the experiments that I benchmarked initially, most of them fully utilized around 4 workers.

Reply:
do we need to tell dask what it's allowed to use?

Reply:
As it's configured now with [`LocalCluster`](https://docs.dask.org/en/latest/setup/single-distributed.html#distributed.deploy.local.LocalCluster) Dask will use as many cores as are available. With 1 worker for each core.

Reply:
So it should just automagically work